modules/sso.php

```diff
@@ -1002,8 +1020,16 @@ public function error_msg_email_already_exists( $message ) {
 	 * @return string
 	 **/
 	public function msg_login_by_jetpack( $message ) {
-
-		$msg = __( sprintf( 'Jetpack authenticates through WordPress.com — to log in, enter your WordPress.com username and password, or <a href="%1$s" target="_blank">visit WordPress.com</a> to create a free account now.', 'http://wordpress.com/signup' ) , 'jetpack' );
+		$msg = sprintf(
+			wp_kses(
+				__(
```

Comment:
Here's some added context and a screenshot of what the UI looks like when this notice shows. It's triggered when a site has removed the default login form.

<img width="401" alt="screen shot 2016-05-20 at 4 41 22 pm" src="https://cloud.githubusercontent.com/assets/1126811/15442896/beec194a-1ea9-11e6-8fb4-13883330a6fb.png">

Based on that, I think we should maybe take out the "Enter your WordPress.com username and password" completely. Or at least mention to enter those details after clicking the button. What do you think?